### DISCUSSION TITLE
Camry LS-Dyna model error: ** RUN KILLED: TOTAL MASS ERROR LIMIT REACHED

### DISCUSSION THREAD

**Discussion by kengwit** [OpenRadioss/OpenRadioss#2279] [Q&A]
Hi,

First, I have seen the discussion here:

https://github.com/orgs/OpenRadioss/discussions/418

but I thought it might be easier if I open a new discussion.

I tried this problem using this build [20240126](https://github.com/OpenRadioss/OpenRadioss/releases/tag/latest-20240126); the commit ID is 192089d762a6fbbc1162b354f88ac48187e98ec5, but I got the same TOTAL MASS ERROR LIMIT REACHED error as described in the discussion above. 

See attached out file. [CamryOpenRadioss_TSC_0001.out.txt](https://github.com/OpenRadioss/OpenRadioss/files/14067604/CamryOpenRadioss_TSC_0001.out.txt)

I have tried running with a smaller time step scale factor (tssfac) of 0.8 and also increased the mass scaled time step (dt2ms) to 0.755e-6, as suggested [here](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/10518559/Camry+Impact+Model+in+LS-DYNA+format) but that did not solve the problem.

Has this issue been actually solved through rigorous testing?

Thanks
Kengwit

**Comment by PaulAltair** [comment]
Hi, I'll have a look at it again, and see if the issue is coming from the same area, I'll let you know if I find a fix.

**Reply by kengwit** [reply]
Thanks

**Reply by PaulAltair** [reply]
I have established that the cause is completely different to before, but no fix yet, still looking

**Reply by kengwit** [reply]
Hi Paul,

Thanks for getting back on this. Will wait for a resolution on this.

**Reply by kengwit** [reply]
Hi any updates on this? I tested this again using the latest release - the same problem still exists.

**Reply by PaulAltair** [reply]
We are still working on this, there are a number of changes to fix other bugs that have had the effect of causing the Camry to no longer work. But it is still on our radar!

**Reply by kengwit** [reply]
Thank you
